<launch>

  <include file="$(find aml_ar)/launch/box_ar.launch">
  </include>

  <node pkg="rosbag" type="play" name="rosbag" args="-s 0 /home/ermanoarruda/Projects/catkin_workspaces/baxter_ws/target_3_new.bag"/>
  
  <node name="extract" pkg="image_view" type="extract_images" respawn="false" output="screen" cwd="ROS_HOME">
    <remap from="image" to="/box_ar/result_ar"/>
  </node>
</launch>